### PR TITLE
Fix UI lint issues

### PR DESCRIPTION
**Pull request by Mogztter**
@adam-cowley I think we should keep the lint task on CI build otherwise we will quickly end-up with a lot of style errors to fix.

### CODE REVIEW

**Comment by Mogztter** [comment]
Build is failing, we need to merge https://github.com/neo4j-documentation/docs-refresh/pull/43 first.

**Comment by Mogztter** [comment]
@adam-cowley We could also add a git hook (using [Husky](https://github.com/typicode/husky)) to automatically lint the code before the commit? If a particular issue cannot be automatically fixed then it would block the commit. That would avoid back-and-forth commits to fix linter issues on a pull request. What do you think?

**Comment by adam-cowley** [comment]
Yeah I like the idea of linting pre-commit.  Just worried that this will slow the user down if you have to wait a few minutes for the content to build before committing.  Maybe a necessary evil?

**Comment by Mogztter** [comment]
> Yeah I like the idea of linting pre-commit. 

Ok :+1: 

> Just worried that this will slow the user down if you have to wait a few minutes for the content to build before committing. Maybe a necessary evil?

`run: npm run lint --prefix ui` only runs the linter on the UI not on the content.
On my machine it takes a few seconds.

Linting the content is another beast and ideally each documentation repository should lint the content before pushing changes.
This project should make sure that the content is valid ("last line of defense") before publishing a new version but that's not something we want to do in a pre-commit hook.

In other words, documentation repositories are responsible for content quality.
This project is only responsible of the documentation UI and the content publication quality. Ultimately, we should make sure that the documentation looks good and that the content is good but we won't fix the content in this repository.
